solidity/contracts/converter/types/standard-pool/StandardPoolConverter.sol

```diff
@@ -37,6 +38,7 @@ contract StandardPoolConverter is
     uint256 private constant AVERAGE_RATE_PERIOD = 10 minutes;
 
     uint256 private __reserveBalances;
+    uint256 private __reserveBalancesProd;
```

Comment:
AFAIK, this should've been added to `ConverterBase` as well. @yudilevi?

Reply:
I think it can wait - we're gonna restructure the inheritance model soon and we might decide to have the standard pool be the parent contract, we'll see.